main_test.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func Test_fixTargetExtension(t *testing.T) {
+	type args struct {
+		targetPath string
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			name: "dir",
+			args: args{"dir"},
+			want: "dir.zip",
+		},
+		{
+			name: "nested_dir",
+			args: args{"/dir/nestedDir"},
+			want: "/dir/nestedDir.zip",
+		},
+		{
+			name: "withExtension",
+			args: args{"dir.zip"},
+			want: "dir.zip",
+		},
+		{
+			name: "file",
+			args: args{"file.text"},
+			want: "file.text.zip",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := fixTargetExtension(tt.args.targetPath); got != tt.want {
+				t.Errorf("fixTargetExtension() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func Test_createConfigsModelFromEnvs(t *testing.T) {
+	tests := []struct {
+		name string
+		want ConfigsModel
+	}{
+		{
+			name: "",
+			want: ConfigsModel{"", ""},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := createConfigsModelFromEnvs(); !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("createConfigsModelFromEnvs() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestConfigsModel_validate(t *testing.T) {
```

Comment:
once, you migrated to the new config handling, remove this test

Reply:
Is there any reason for that? 
Because I don't get it why should we remove this test.